<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(11, 11, 11);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="fontFrame">
        <property name="minimumSize">
         <size>
          <width>426</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>38</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: rgb(85, 170, 255);
background-color: rgb(85, 170, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSpinBox" name="fontSize">
           <property name="font">
            <font>
             <family>Segoe UI</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="value">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Segoe UI</family>
            </font>
           </property>
           <property name="text">
            <string>Font Size</string>
           </property>
           <property name="buddy">
            <cstring>fontSize</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="bold">
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>13</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
            </font>
           </property>
           <property name="text">
            <string>Bold</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="underline">
           <property name="font">
            <font>
             <family>Segoe UI</family>
            </font>
           </property>
           <property name="text">
            <string>Underline</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="italics">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
            </font>
           </property>
           <property name="text">
            <string>Italics</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="drawFrame">
        <property name="minimumSize">
         <size>
          <width>426</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>38</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 255, 127);
border-color: rgb(85, 170, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="paintBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Paint</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="brushBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-color: rgb(255, 0, 0);
alternate-background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);

</string>
           </property>
           <property name="text">
            <string>Brush Size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="colorBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-color: rgb(255, 0, 0);
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Brush Color</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(255, 0, 0);
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>underline</sender>
   <signal>toggled(bool)</signal>
   <receiver>textEdit</receiver>
   <slot>setFontUnderline(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
